pkg/kubelet/config/file.go

```diff
@@ -200,3 +201,21 @@ func (s *sourceFile) replaceStore(pods ...*v1.Pod) (err error) {
 	}
 	return s.store.Replace(objs, "")
 }
+
+func checkPlainTextFile(file string) (bool, error) {
+	out, err := exec.Command("file", "--mime-type", file).CombinedOutput()
```

Comment:
I suggest using `out, err := exec.Command("mimetype", "--output-format=%m", file).CombinedOutput()` instead to avoid extra output split and analysis.